src/templates/admin/elements/submit/file.html

```diff
@@ -8,17 +8,11 @@ <h5><i class="fa fa-upload">&nbsp;</i>{% trans "Add New File" %}</h5>
         </div>
         <div class="card-section">
             <small>{% include "elements/forms/errors.html" with form=form %}</small>
-            {% if modaltype == 'data' %}
-                <p>{% trans "Please supply any original figure files or supporting information. The journal urges you to submit this document to a repository such as Figshare so that a DOI can be assigned to it, but this is not required." %}</p>
-            {% else %}
-                {% if not journal_settings.general.limit_manuscript_types %}
-                    <p>{% trans "Please upload your manuscript file, it should be in PDF, RTF, ODT or DOC/X format. LaTeX may be accepted later in the process but a simple, readable file is required at this stage." %}</p>
-                {% else %}
-                    <p>{% trans "Please upload your manuscript file, it should be in RTF, ODT or DOC/X format. LaTeX may be accepted later in the process but a simple, readable file is required at this stage." %}</p>
+            {% if modaltype == 'manuscript' %}
+                {% if journal_settings.general.limit_manuscript_types %}
+                    <p>{% trans "File types limited to Doc, Docx, RTF, and ODT." %}</p>
```

Comment:
Changing this text would require requesting new translations to our contributors, which makes me wonder if it is at all necessary since the text is not very different from what we had previously

Reply:
The change is to fix a grammatical "error" that many editors might find distracting. We don't copyedit things before sending them for translation, do we?

Reply:
We do not.

In my opinion we should only use variable names here rather than the actual text and store the english text in the .po file instead. (the .po files is where we store the translation strings)

Alternatively, you can change the name of this string (`msgid`) on the PO file for the other languages so that we can preserve the translations that we currently have (e.g.: https://github.com/BirkbeckCTP/janeway/blob/84a0d087926ef4f779bc37c6d78ad15df5f3c7f2/src/core/locales/nl/LC_MESSAGES/django.po#L812)